Tests/SwiftGenKitTests/ColorsCLRFileTests.swift

```diff
@@ -10,30 +10,30 @@ import SwiftGenKit
 
 class ColorsCLRFileTests: XCTestCase {
   func testEmpty() {
-    let parser = ColorsCLRFileParser()
+    let parser = ColorsFileParser()
 
     let result = parser.stencilContext()
     XCTDiffContexts(result, expected: "empty.plist", sub: .colors)
   }
 
   func testFileWithDefaults() throws {
-    let parser = ColorsCLRFileParser()
+    let parser = ColorsFileParser()
```

Comment:
Shouldn't we keep `ColorsCLRFileParser` here and in other `Colors*FileTests`?
Because if we do this, I have the feeling we are testing two things:
1. the `ColorsCLRFileParser` part behind `ColorsFileParser`
2. and the "parser routing" in `ColorsFileParser` which select the correct parser to use

It would be nice to split the specific parsing tests with the tests to select the correct file parser in `ColorsFileParser`.
In fact, the registration mechanism suggested by @AliSoftware would allow to test this last part quite easily IMO :)